From 4a799f42cc93ed6678e0c6ba5b274890d09505c1 Mon Sep 17 00:00:00 2001
From: Cody Xie <Cody.Xie@rock-chips.com>
Date: Wed, 23 Apr 2014 19:08:51 +0800
Subject: [PATCH] Revert "rk3288-p977 : diable 5v_power_en to fix the issue
 that usb device cannot be detected"

This reverts commit 4680277b7a6cbb000eb270d99a69d743a422e05d.
---
 arch/arm/boot/dts/rk3288-p977.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rk3288-p977.dts b/arch/arm/boot/dts/rk3288-p977.dts
index a5fa5f03193c..b5ecfe04cc07 100755
--- a/arch/arm/boot/dts/rk3288-p977.dts
+++ b/arch/arm/boot/dts/rk3288-p977.dts
@@ -11,7 +11,7 @@
 	5v_power_en {
         compatible = "5v_en";
 		5ven,pin =<&gpio7 GPIO_B4 GPIO_ACTIVE_HIGH>;	// 5v power for otg and hdmi
-		status = "disabled";	
+		status = "okay";	
 	};
 
 
-- 
2.35.3

